components/Reference/columns.tsx

```diff
@@ -2,9 +2,13 @@ import { Row } from 'react-table'
 
 import { StackBox } from 'components/ui'
 
-const filter = (rows: Row[], id: string, filterValue: string) => {
-  const re = new RegExp(`${filterValue}`, 'i')
```

Comment:
This is just a case insenctive matching pattern - instead of using regex which can cause issues when users write invalid patterns, we use vanilla javascript 😄